### PR TITLE
added some responsive changes

### PR DESCRIPTION
**Pull request by skyspace88**
here are the changes :)

### CODE REVIEW

**Comment by skyspace88** [comment]
not sure why it keeps creating a second branch when pushing from the command line, will need to look more into that, my apologies

File: gameBase/src/css/styles.css [hunk 1]
```diff
@@ -11,8 +11,8 @@ body {
 }
 
 .memory-game {
```
Comment (GeraAlcantara) [inline gameBase/src/css/styles.css:13]: try to add 
width: 100%;
  height: calc(100% / 4 * 3); // control aspect ratio 
  padding: 10px;
  max-height: 640px;
  max-width: 640px;

Comment (skyspace88) [reply gameBase/src/css/styles.css:13]: in the media query?

Comment (GeraAlcantara) [reply gameBase/src/css/styles.css:13]: nope directly in the class .memory-game
the max-width and max-height will prevent that bigger screens expand 100%

Comment (skyspace88) [reply gameBase/src/css/styles.css:13]: oh i see!!! added, can you close this pull request so i can make another one

**Comment by GeraAlcantara** [comment]
@dreymotoh ?
you close the pull request before any merge

**Comment by skyspace88** [comment]
oh, i thought i had to send another pull request since i added changes to the files, thats what another maintainer was making me do, my apologies

**Pull request closed without merging**